ui/src/main/js/view/templates/index.js

```diff
@@ -63,6 +63,12 @@ function registerHBSHelper(name, helper) {
     handlebars.registerHelper(name, helper);
 }
 
+registerHBSHelper('breaklines', function(text) {
+    text = handlebars.escapeExpression(text);
+    text = text.replace(/(\r\n|\n|\r)/gm, '<br>');
```

Comment:
BTW there is a utility in `Functions` (or is it `Util`?) which does just this, except better (considering good break points etc.); but it is not available for JavaScript code.